.github/ISSUE_TEMPLATE/libmamba-bug-report.yml

```
@@ -0,0 +1,86 @@
+
+        There are also already a lot of open issues, so please take 2 minutes and search through existing ones to see if what you are experiencing already exists
+
+        Thanks for helping conda be amazing. We are nothing without people like you helping build a better community 💐!
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched for other issues and couldn't find a solution or duplication
+          required: true
+        - label: I looked at the [README.md](https://github.com/conda-incubator/conda-libmamba-solver) and didn't see anything to help
+          required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: And what should have happened instead? This really helps us triage quicker and greatly increases the chance that we can get around to solve your issue
+      placeholder: Tell us what you see!
+      value: "A wild Caterpie appeared!"
+    validations:
+      required: true
+  - type: textarea
+    id: info
+    attributes:
+      label: Conda info
+      description: |
+        Let's get the info on your conda setup. Run the following command from your command line and paste the output
+
+        ```bash
+        conda info
+        ```
+      render: shell
+  - type: textarea
+    id: config
+    attributes:
+      label: Conda config
+      description: |
+        Let's see what your conda config looks like. Run the following command from your command line and paste the output
+
+        ```bash
+        conda config --show-sources
+        ```
+      render: shell
+  - type: textarea
+    id: list
+    attributes:
+      label: Conda list
+      description: |
+        These are the packages installed in your conda environment and the conda channels used. They tell us where you are pulling packages from. Make sure you are in the environment where \n you are seeing the issues. Run the following command from your command line and paste the output
+
+        ```bash
+        conda list --show-channel-urls
+        ```
+      render: shell
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Context
+      description: Add any additional context information or screenshots you think are useful.
+      
+  - type: markdown
+    attributes:
+      value: |
+       ## Important: 
+       Please attach the log files in a comment to this issue after you have used the `Submit new issue` button. The log files should be found on your computer at `<install path of anaconda>\envs\libmamba\pkgs\.logs\`. You will also see this exact path as it relates to your system on the command line after running anything with `--experimental-solver=libmamba`. 
```

Looks like any `textarea` without the `render` attribute defined will accept attachments (see [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#:~:text=When%20this%20key%20is%20provided%2C%20the%20text%20area%20will%20not%20expand%20for%20file%20attachments%20or%20Markdown%20editing.)):

```suggestion      
  - type: textarea
    attributes:
      label: Log file
      description: |
        Please attach the associated log file (drag and drop the file into the textbox
        below). The log file can be found at `$CONDA_PREFIX\pkgs\.logs\`. The
        exact path is also printed to the command line when running commands with
        `--experimental-solver=libmamba`.
```

Assuming this all works as expected I'd also swap this "Log file" block with the "Additional Context" block.

---

Nice find, yeah that will be a cleaner way to do it.

I do think that the "CONDA_PREFIX" part is really confusing to an average user though, saying it in a more common language I think would be clearer.

---

Yeah I agree with `$CONDA_PREFIX` being non-obvious but I don't think there's a good way to broadly encapsulate any environment name/path otherwise?

---

Actually just adding a quick explanation of what it means will be good. I'll add that.

---

Could also have "the environment where libmamba is installed". It might not matter that much as the path would be shown. We can see if we need to reword if people are having issues.

---

It's actually "a `.logs` directory under your package cache directory", which can be `$CONDA_ROOT/pkgs`, `$CONDA_PREFIX/envs/<env>/pkgs` or `~/.conda/pkgs`, depending on the configuration and directory permissions.

---

Ah, I didn't know that, since there are 3 different places, asking the user to figure out where it may be located is probably too much.

I'd be in favor of dropping the `.logs` instructions and only specifying that the path is printed to the command line.

---

Yes it's a bit weird. It's the "first writable cache directory" (which might also change across runs!), but it uses the same logic as conda. I agree that we should just refer to the blurb posted at the beginning.